circleci.tf

```diff
@@ -36,14 +36,44 @@ variable "services_instance_type" {
 
 variable "builder_instance_type" {
     description = "instance type for the builder machines.  We recommend a r3 instance"
-    default = "r3.2xlarge"
+    default = "r3.4xlarge"
+}
+
+variable "circle_container_cpus" {
+    description = "CPUs allocated per container"
+    default = "2"
+}
+
+variable "circle_container_image_id" {
+    description = "Container image ID"
+    default = "circleci-precise-container_0.0.1338"
+}
+
+variable "circle_container_memory_limit" {
+    description = "Memory limit per container"
+    default = "4G"
+}
+
+variable "circle_num_containers" {
+    description = "Number of builder containers"
+    default = "7"
```

Comment:
Matches the recommendation in https://circleci.com/docs/enterprise/config/#supported-builder-instance-types